File: core/src/backendServer/models/ShotgunPickup.js
const Pickup = require('./Pickup');
const constants = require('../settings/constants');
const ProjectileType = require('./ProjectileType');

class ShotgunPickup extends Pickup {
    constructor(x, y, id, ammunition = constants.SHOTGUN_BULLETS_IN_CHAMBER) {
        super(x, y, constants.SHOTGUN_SPRITE_WIDTH, constants.SHOTGUN_SPRITE_HEIGHT, ProjectileType.SHOTGUN);
    }
}

module.exports = ShotgunPickup

File: core/src/com/mygdx/game/model/pickup/ShotgunPickup.kt
package com.mygdx.game.model.pickup

import com.badlogic.gdx.graphics.Texture
import com.mygdx.game.model.pickup.Pickup
import com.mygdx.game.settings.SHOTGUN_SPRITE_HEIGHT
import com.mygdx.game.settings.SHOTGUN_SPRITE_WIDTH

class ShotgunPickup(
        x: Float = 0f,
        y: Float = 0f,
        texture: Texture) : Pickup(
        x,
        y,
        SHOTGUN_SPRITE_WIDTH,
        SHOTGUN_SPRITE_HEIGHT,
        texture
)